app/services/mhv_logging_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+require 'mhv_logging/client'
+class MHVLoggingService
+  def self.login(current_user)
+    # If login has already been submitted, do nothing
+    return false if current_user.mhv_last_signed_in || current_user.mhv_correlation_id.nil?
```

Comment:
total nit, but maybe reverse the order of these? if there is no MHV ID, it doesn't need to evaluate whether the user has signed into MHV.